README.md

```diff
@@ -6,38 +6,81 @@ the Transmission BitTorrent client, using its HTTP RPC protocol.
 
 # DEPENDENCIES
 
-The following packages are required dependencies (debian/ubuntu).
+### Runtime Required Dependencies
+The following packages are required dependencies:
 
-```bash
-libgtk-3-dev automake autoconf gcc libgeoip-dev gettext autoconf-archive libappstream-dev appstream-util libcurl4-openssl-dev libjson-glib-dev
+ - gtk >= 3.16
+ - glib >= 2.44 (including gio and gthread)
+ - json-glib >= 0.8
+ - libcurl
+ - GNU gettext
+
+An example command for getting the packages required for an Ubuntu/Debian system:
+
+```
+# apt-get install libgtk-3-dev libgeoip-dev gettext libcurl4-openssl-dev libjson-glib-dev
 ```
 
-# BUILDING
+### Runtime Optional Dependencies
+The following packages are optinal dependencies
+
+ - libmrss >= 0.18, RSS feed support
+ - libproxy, HTTP proxy support
+ - libgeoip, country of origin of peers
+ - libnotify, desktop notifications support
+ - libappindicator, Application tray support
 
-Optionally install `libgeoip` and its headers to see the country
-origin of peers.
+### Build Dependencies
+Transmission Remote Gtk uses meson for its build system. A relatively new version of
+meson is required.
 
+ - meson >= 0.59.0
+ - ninja
+ - gcc >= 4.5
+ - appstream-util (optional, desktop/appdata validation)
+ - pod2man (optional, manpage generation)
 
-##  Building a transmission-remote-gtk release from the command line:
+An example of getting the build dependencies for an Ubuntu/Debian system:
 
-```bash
-$ tar zxfv transmission-remote-gtk-1.2.tar.gz
-$ cd transmission-remote-gtk-1.2
-$ ./configure
-$ make
-# make install
+```
+# apt-get install gcc python3 meson ninja-build libappstream-dev appstream-util perl
 ```
 
-##  Building from git clone:
+**NOTE:** Ubuntu and other stable or LTS distros may have outdated versions of meson
+that will not work for compiling transmission-remote-gtk. Meson can also be
+installed via pip:
 
-```bash
-$ git clone https://github.com/transmission-remote-gtk/transmission-remote-gtk.git
-$ ./autogen.sh
-$ ./configure
-$ make
-# make install
+```
+# apt-get install python3-pip
+$ pip install meson ninja
+```
+
+
+# BUILDING
+
+transmission-remote-gtk can be build from a release tarball or by cloning the
+repository and building from that directly.
+
+### From a tarball
+The following commands will build transmission-remote-gtk on the command line
+from a release tarball.
+```
+$ tar zxfv transmission-remote-gtk-${VERSION}.tar.gz
+$ cd transmission-remote-gtk-${VERSION}
+$ meson build
+$ meson compile -C build
+# meson install -C build
 ```
 
+###  From git clone
+The following commands will build transmission-remote-gtk after a git clone
+```
+$ git clone https://github.com/transmission-remote-gtk/transmission-remote-gtk.git
+$ cd transmission-remote-gtk
+$ meson build
+$ meson compile -C build
+# meson install -C build
```

Comment:
A lot of duplication here.

Honestly only document building from git, and mention tarballs are available for releases.